src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -151,11 +158,18 @@ private static void usage() {
         + "   to start a server, you can optionally password protect the game using triplea.server.password=foo");
   }
 
-  public static void main(final String[] args) {
+  public static void main(String[] args) {
+    if( args == null ) {
+      args = new String[0];
+    }
+    if( exceptionHandler == null ) {
+      exceptionHandler = new ErrorHandler();
+    }
+    Thread.setDefaultUncaughtExceptionHandler( exceptionHandler);
+
     setupLogging();
     Console.getConsole().displayStandardError();
     Console.getConsole().displayStandardOutput();
-    System.setProperty("sun.awt.exception.handler", ErrorHandler.class.getName());
```

Comment:
Wouldn't this apply to all threads, while Thread.setDefaultUncaughtExceptionHandler only apply to the current thread?